pages/workflow/validate.mdx

```diff
@@ -0,0 +1,176 @@
+---
+layout: 'guide'
+title: '5. Validate implementation'
+abstract: ''
+---
+
+import Link from '../../components/Link';
+
+Avo provides a number of tools that help you verify that the events you’ve implemented are correct according to how you’ve defined them in your Avo Tracking Plan.
+
+Avo’s validation functionalities are built on top of two product offerings, <Link passHref href="/data-design/start-using-inspector"><a>Avo Inspector</a></Link> and <Link passHref href="/implementation/avo-functions-overview"><a>Avo Functions</a></Link>.
+
+# Validate implementation with Avo Inspector
+
+For every source that has the Inspector installed, you can validate if the implementation is according to the tracking plan using the Inspector dashboard and/or the Inspector implementation in your tracking plan.
+
+If you don’t have the Inspector installed yet, you can learn more about how to get started with the Inspector here:
+
+> [Use Inspector to improve your tracking plan health](https://www.avo.app/docs/data-design/start-using-inspector)
+
+
+**Validate with the Inspector dashboard**
```

Comment:
We should use ## or ### for all subtitles here so they show up in the page overview

![Screenshot 2022-07-18 at 17 58 47](https://user-images.githubusercontent.com/3513968/179573453-6ab24a49-203c-4f26-b2fc-3c49e13ee55c.png)

```suggestion
### Validate with the Inspector dashboard
```